layers/+lang/go/packages.el

```diff
@@ -145,3 +146,12 @@
 
 (defun go/post-init-helm-gtags ()
   (spacemacs/helm-gtags-define-keys-for-mode 'go-mode))
+
+(defun go/init-godoctor ()
+  (use-package godoctor
+    :init
+    (spacemacs/set-leader-keys-for-major-mode 'go-mode
+      "dr" 'godoctor-rename
+      "de" 'godoctor-extract
+      "dt" 'godoctor-toggle
+      "dd" 'godoctor-godoc)))
```

Comment:
Are these refactoring? If so, the refactoring binding is [`SPC m r`](https://github.com/syl20bnr/spacemacs/blob/master/doc/CONVENTIONS.org#refactoring)